docs/en/sql-reference/00-sql-reference/10-data-types/43-data-type-variant.md

```diff
@@ -102,11 +102,11 @@ Result:
 └────────────┴─────────────┘
 ```
 
-## Get by field name
+### Accessing by Field Name
```

Comment:
Please add note that `Field Name` is case sensitive in accessing.

1. `preferences:"setting"` 
2. `preferences:"setting"` 
3. `preferences:"Setting"` 

1 is equal to 2, but 3 is not.